spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/store/channel/DeleteReturningPostgresChannelMessageStoreQueryProvider.java

```diff
@@ -0,0 +1,76 @@
+package org.springframework.integration.jdbc.store.channel;
+
+/**
+ * @author Johannes Edmeier
+ *
+ * @since 6.2
+ */
+public class DeleteReturningPostgresChannelMessageStoreQueryProvider extends PostgresChannelMessageStoreQueryProvider {
```

Comment:
Why do we still need this?
Why we cannot override all those methods in the `PostgresChannelMessageStoreQueryProvider` directly since we have already introduced that ` isSingleStatementForPoll()`?

Reply:
done